ERROR - 2023-04-03 04:41:44 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'roor'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('127.0.0.1', 'roor', Object(SensitiveParameterValue), 'library_rc', 3306, '', 0)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#6 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-04-03 04:41:44 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `librarylogin` 
												  WHERE `IL_Key`=\'admin902022\' 
												  AND `IL_Pass`=\'587367fb08ec35e3997e54c75701741e\' 
												  AND `IL_Status`=\'1\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-03 04:42:00 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'roor'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('127.0.0.1', 'roor', Object(SensitiveParameterValue), 'library_rc', 3306, '', 0)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#6 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-04-03 04:42:00 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `librarylogin` 
												  WHERE `IL_Key`=\'\' 
												  AND `IL_Pass`=\'d41d8cd98f00b204e9800998ecf8427e\' 
												  AND `IL_Status`=\'1\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-03 04:42:32 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'roor'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('127.0.0.1', 'roor', Object(SensitiveParameterValue), 'library_rc', 3306, '', 0)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#6 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-04-03 04:42:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `librarylogin` 
												  WHERE `IL_Key`=\'\' 
												  AND `IL_Pass`=\'d41d8cd98f00b204e9800998ecf8427e\' 
												  AND `IL_Status`=\'1\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-03 04:42:32 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'roor'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('127.0.0.1', 'roor', Object(SensitiveParameterValue), 'library_rc', 3306, '', 0)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#6 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-04-03 04:42:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `librarylogin` 
												  WHERE `IL_Key`=\'\' 
												  AND `IL_Pass`=\'d41d8cd98f00b204e9800998ecf8427e\' 
												  AND `IL_Status`=\'1\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-03 04:42:32 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'roor'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('127.0.0.1', 'roor', Object(SensitiveParameterValue), 'library_rc', 3306, '', 0)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#6 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-04-03 04:42:32 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `librarylogin` 
												  WHERE `IL_Key`=\'\' 
												  AND `IL_Pass`=\'d41d8cd98f00b204e9800998ecf8427e\' 
												  AND `IL_Status`=\'1\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-03 04:42:33 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'roor'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('127.0.0.1', 'roor', Object(SensitiveParameterValue), 'library_rc', 3306, '', 0)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#6 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-04-03 04:42:33 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `librarylogin` 
												  WHERE `IL_Key`=\'\' 
												  AND `IL_Pass`=\'d41d8cd98f00b204e9800998ecf8427e\' 
												  AND `IL_Status`=\'1\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-03 04:42:33 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'roor'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('127.0.0.1', 'roor', Object(SensitiveParameterValue), 'library_rc', 3306, '', 0)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#6 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-04-03 04:42:33 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `librarylogin` 
												  WHERE `IL_Key`=\'\' 
												  AND `IL_Pass`=\'d41d8cd98f00b204e9800998ecf8427e\' 
												  AND `IL_Status`=\'1\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-03 04:42:40 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'roor'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('127.0.0.1', 'roor', Object(SensitiveParameterValue), 'library_rc', 3306, '', 0)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#6 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-04-03 04:42:40 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `librarylogin` 
												  WHERE `IL_Key`=\'admin902022\' 
												  AND `IL_Pass`=\'587367fb08ec35e3997e54c75701741e\' 
												  AND `IL_Status`=\'1\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
ERROR - 2023-04-03 04:44:08 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'roor'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('127.0.0.1', 'roor', Object(SensitiveParameterValue), 'library_rc', 3306, '', 0)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#6 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in D:\XAMPP\htdocs\library_regina\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\XAMPP\htdocs\library_regina\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\XAMPP\htdocs\library_regina\app\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#3 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
#4 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
#5 D:\XAMPP\htdocs\library_regina\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\XAMPP\htdocs\library_regina\index.php(64): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2023-04-03 04:44:08 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserLogin.php(14): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `librarylogin` 
												  WHERE `IL_Key`=\'admin902022\' 
												  AND `IL_Pass`=\'587367fb08ec35e3997e54c75701741e\' 
												  AND `IL_Status`=\'1\'')
 3 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\UserLogin->LibraryLogin()
 4 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserLogin))
 5 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-04-03 04:48:17 --> Undefined variable $CountLibrary
in FCPATH\public\database\pdo_library.php on line 62.
 1 FCPATH\public\database\pdo_library.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $CountLibrary', 'D:\\XAMPP\\htdocs\\library_regina\\public\\database\\pdo_library.php', 62)
 2 FCPATH\public\database\class_library.php(1067): count_library->__construct()
 3 FCPATH\public\admin_mod\books_register.php(102): ManagementAuthor->__construct('-', '-', 'read')
 4 APPPATH\Views\admin\administrator.php(110): include('D:\\XAMPP\\htdocs\\library_regina\\public\\admin_mod\\books_register.php')
 5 SYSTEMPATH\View\View.php(213): include('D:\\XAMPP\\htdocs\\library_regina\\app\\Views\\admin\\administrator.php')
 6 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 7 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('admin/administrator', [], true)
 8 APPPATH\Controllers\Admin.php(5): view('admin/administrator')
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Admin->index()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-04-03 04:48:28 --> Undefined variable $CountLibrary
in FCPATH\public\database\pdo_library.php on line 62.
 1 FCPATH\public\database\pdo_library.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $CountLibrary', 'D:\\XAMPP\\htdocs\\library_regina\\public\\database\\pdo_library.php', 62)
 2 FCPATH\public\database\class_library.php(1067): count_library->__construct()
 3 FCPATH\public\admin_mod\books_register.php(102): ManagementAuthor->__construct('-', '-', 'read')
 4 APPPATH\Views\admin\administrator.php(110): include('D:\\XAMPP\\htdocs\\library_regina\\public\\admin_mod\\books_register.php')
 5 SYSTEMPATH\View\View.php(213): include('D:\\XAMPP\\htdocs\\library_regina\\app\\Views\\admin\\administrator.php')
 6 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 7 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('admin/administrator', [], true)
 8 APPPATH\Controllers\Admin.php(5): view('admin/administrator')
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Admin->index()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-04-03 04:50:02 --> Undefined variable $CountLibrary
in FCPATH\public\database\pdo_library.php on line 62.
 1 FCPATH\public\database\pdo_library.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $CountLibrary', 'D:\\XAMPP\\htdocs\\library_regina\\public\\database\\pdo_library.php', 62)
 2 FCPATH\public\database\class_library.php(1067): count_library->__construct()
 3 FCPATH\public\admin_mod\books_register.php(102): ManagementAuthor->__construct('-', '-', 'read')
 4 APPPATH\Views\admin\administrator.php(110): include('D:\\XAMPP\\htdocs\\library_regina\\public\\admin_mod\\books_register.php')
 5 SYSTEMPATH\View\View.php(213): include('D:\\XAMPP\\htdocs\\library_regina\\app\\Views\\admin\\administrator.php')
 6 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 7 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('admin/administrator', [], true)
 8 APPPATH\Controllers\Admin.php(5): view('admin/administrator')
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Admin->index()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-04-03 04:50:24 --> Undefined variable $CountLibrary
in FCPATH\public\database\pdo_library.php on line 62.
 1 FCPATH\public\database\pdo_library.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $CountLibrary', 'D:\\XAMPP\\htdocs\\library_regina\\public\\database\\pdo_library.php', 62)
 2 FCPATH\public\database\class_library.php(1067): count_library->__construct()
 3 FCPATH\public\admin_mod\books_register.php(102): ManagementAuthor->__construct('-', '-', 'read')
 4 APPPATH\Views\admin\administrator.php(110): include('D:\\XAMPP\\htdocs\\library_regina\\public\\admin_mod\\books_register.php')
 5 SYSTEMPATH\View\View.php(213): include('D:\\XAMPP\\htdocs\\library_regina\\app\\Views\\admin\\administrator.php')
 6 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 7 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('admin/administrator', [], true)
 8 APPPATH\Controllers\Admin.php(5): view('admin/administrator')
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Admin->index()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Admin))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(64): CodeIgniter\CodeIgniter->run()
